homeassistant/components/trello/strings.json

```diff
@@ -0,0 +1,25 @@
+{
+  "config": {
+    "step": {
+      "user": {},
+      "creds": {
+        "data": {
+          "api_key": "[%key:common::config_flow::data::api_key%]",
+          "api_token": "[%key:common::config_flow::data::api_token%]"
+        }
+      },
+      "boards": {
+        "data": {
+          "board_ids": "Board names"
```

Comment:
I'd go with "Boards" as you're selecting boards and not board names.  (You're showing board names, but you're not selecting them ;))

Reply:
Makes sense. Will make this change.